### PR TITLE
KOGITO-3971: Empty results card

### PR DESCRIPTION
**Pull request by Sara4994**
This PR is intended to add table headers/controls when no jobs found in Jobs section.

JIRA link: https://issues.redhat.com/browse/KOGITO-3971

### CODE REVIEW

File: ui-packages/packages/common/src/components/Molecules/EmptyResultsCard/EmptyResultsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  Bullseye
+} from '@patternfly/react-core';
+import { SearchIcon } from '@patternfly/react-icons';
+
+interface IOwnProps {
+  titleText: string;
+  bodyText: string;
+}
+
+const EmptyResultsCard: React.FC<IOwnProps> = ({ titleText, bodyText }) => {
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/components/Molecules/EmptyResultsCard/EmptyResultsCard.tsx]: Add OUIA component attributes please.

Comment (Sara4994) [reply ui-packages/packages/common/src/components/Molecules/EmptyResultsCard/EmptyResultsCard.tsx]: @jstastny-cz done :+1:

**Review by jstastny-cz — APPROVED** [review]

**Comment by Sara4994** [comment]
jenkins please retest this

**Review by AjayJagan — APPROVED** [review]
LGTM

**Comment by Sara4994** [comment]
jenkins please retest this

**Comment by nmirasch** [comment]
@Sara4994  I see a line that separates the filter area from the results at Process instances
Selection_447

Could we possible to add that line here too when there aren't results ?

Selection_448

**Comment by Sara4994** [comment]
> @Sara4994 I see a line that separates the filter area from the results at Process instances
> Selection_447
> 
> Could we possible to add that line here too when there aren't results ?
> 
> Selection_448

@nmirasch done :+1:

**Comment by Sara4994** [comment]
jenkins please retest this

**Comment by nmirasch** [comment]
@Sara4994 first question is: Why we need another component for emptyResultCard, Could we use 'KogitoEmptyState' that is so generic to include this use case?
The second thing, taking a look on Patternfly emptyState https://www.patternfly.org/2020.04/documentation/react/components/table#empty-state it seems the columns header are visible. I know we don't have header columns at Process instances and there were the line but in the table cases we need to include the headers to follow patternfly, Wdyt?

**Comment by Sara4994** [comment]
> @Sara4994 first question is: Why we need another component for emptyResultCard, Could we use 'KogitoEmptyState' that is so generic to include this use case?
> The second thing, taking a look on Patternfly emptyState https://www.patternfly.org/2020.04/documentation/react/components/table#empty-state it seems the columns header are visible. I know we don't have header columns at Process instances and there were the line but in the table cases we need to include the headers to follow patternfly, Wdyt?

@nmirasch As discussed, table header will be made visible for empty states once we have the table on the process list page as well.

**Comment by nmirasch** [comment]
@Sara4994 overall looks good and works fine. The only thing I found is about the empty state with 'No filter applied' that shows the load more button
image

**Review by nmirasch — APPROVED** [review]
Thanks @Sara4994!

**Pull request merged**